starwars/src/components/StarWarsData.js

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import StarWarsCard from "./StarWarsCard";
+
+export function StarWarsData() {
+  const [starWarsChars, setStarWarsChars] = useState([]);
+  const [page, setPage] = useState([]);
+
+  useEffect(() => {
+    console.log("component mounted");
+    axios
+      .get("https://swapi.co/api/people/?page=1")
```

Comment:
You were so close! Another way you could have done the pagination stretch is to manipulate the number of the page in the url's query. Awesome effort though